pkg/resources/marketplace/manager.go

```diff
@@ -75,14 +76,14 @@ func (m *Manager) InstallOperator(ctx context.Context, serverClient k8sclient.Cl
 	}
 	err = serverClient.Create(ctx, og)
 	if err != nil && !k8serr.IsAlreadyExists(err) {
-		logrus.Infof("error creating operator group")
+		log.Info("error creating operator group")
```

Comment:
could this be made into an log.Error message?

Reply:
done